sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientTests.cs

```diff
@@ -98,6 +99,23 @@ public void ConstructorRequiresTheOptions()
             Assert.Throws<ArgumentNullException>(() => new FormRecognizerClient(endpoint, keyCredential, null));
         }
 
+        [Test]
+        public async Task FormRecognizerClientThrowsWithInvalidEndpoint()
```

Comment:
Consider changing the name of the method to `FormRecognizerClientThrowsWithInvalidUri`

Reply:
hmmm not sure about that anymore

Reply:
What about `FormRecognizerClientThrowsWithNonExistingResourceEndpoint`?

Reply:
I like it!